exercises/subprograms/challenge_119.py

```diff
@@ -0,0 +1,68 @@
+"""
+Define a subprogram that will ask the user to pick a low and a high number,
+and then generate a random number between those two values and store it in
+a variable called  “comp_num”.
+
+Define another  subprogram that will  give the instruction “I am  thinking
+of a number…” and then ask the user to guess the number they are thinking of.
+
+Define a third subprogram that will check to see if the comp_num is the same
+as the user’s guess. If it is, it should display the message “Correct, you
+win”, otherwise it should keep looping, telling the user if they are too low or
+too high and asking them to guess again until they guess correctly.
+"""
+
+import random
+import common_functions
+
+
+def ask_for_user_numbers():
+    while True:
+        num_low = common_functions.get_user_input_integer(
+            "Please, enter the minimum number: "
+        )
+        num_high = common_functions.get_user_input_integer(
+            "Please, enter the maximum number: "
+        )
+
+        if num_low < num_high:
+            comp_num = random.randrange(num_low, num_high)
+            return comp_num
+
+        print(
+            "Please ensure that the minimum number is smaller than the maximum number"
+        )
+
+
+def prompt_for_guess():
+    print("\nI am thinking of a number… ")
+    guess = common_functions.get_user_input_integer(
+        "Please, enter the number I'm thinking of: "
+    )
+    return guess
+
+
+def check_user_choice(computer_choice, user_guess):
+    while True:
+        if computer_choice == user_guess:
+            print("Correct, you win!")
+            return
+
+        if user_guess > computer_choice:
```

Comment:
Did you use `if` here on purpose instead of an `elif`? What's the advantage of each?